src/main/java/com/synopsys/integration/detect/lifecycle/run/DetectRun.java

```diff
@@ -58,6 +60,9 @@ public void run(BootSingletons bootSingletons) {
             UniversalStepRunner stepRunner = new UniversalStepRunner(operationRunner, stepHelper); //Product independent tools
             UniversalToolsResult universalToolsResult = stepRunner.runUniversalTools();
 
+            // test for some image scan results and throw exception if not found.
```

Comment:
I think it might be worthwhile to add to the comment that if this is not an image scan then we won't throw the exception. It currently reads like you have to have something.

Reply:
I was just reading that comment when I thought "it seems a bit vague"

Reply:
done